src/media/js/main.js

```diff
@@ -61,7 +62,7 @@ define('main', [
     });
 
     z.page.on('navigate', function(e, url) {
-        if ([urls.reverse('login'), urls.reverse('fxa_authorize')].indexOf(url)) {
+        if ([urls.reverse('login'), urls.reverse('core/fxa_authorize')].indexOf(url)) {
```

Comment:
This actually doesn't infinitely recurse since it triggers a `divert` and this only runs on `navigate`. So it just works out. I fixed it though.